geometry_msgs/PointStamped point
---
